R/oauth-init.R

```diff
@@ -31,11 +31,11 @@ init_oauth1.0 <- function(endpoint, app, permission = NULL,
   authorize_url <- modify_url(endpoint$authorize, query = list(
     oauth_token = token,
     permission = "read"))
-  verifier <- oauth_listener(authorize_url, is_interactive)[[1]]
+  verifier <- oauth_listener(authorize_url, is_interactive)$oauth_verifier
```

Comment:
Yeah it was changed with https://github.com/hadley/httr/commit/7261a525ede1beb9880d1ce2e2009d27c8e8bef7#commitcomment-17702567, unfortunatly subsetting by position breaks when authenticating with flickr at least. I don't know if they are not returning the proper response or if it needs to be reverted.